.github/workflows/check_for_deleted_assets.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           echo ${{ env.PR_NUMBER }} >> $artifactName
           echo ${{ steps.set-deleted-files-count.outputs.result }} >> $artifactName
       - name: Upload the deleted assets file to artifacts
-        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2 https://github.com/actions/upload-artifact/commit/0b7f8abb1508181956e8e162db84b466c27e18ce
+        uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808 # v4.3.3
```

Comment:
lets just drop the versions since we want this to autoupdate in the future

Reply:
I think dependabot will actually update the comment as long as it contains a version. Example from a previous PR here: https://github.com/aws-amplify/docs/pull/7507/files